src/main/java/org/jenkinsci/plugins/workinghours/model/DateType.java

```diff
@@ -0,0 +1,6 @@
+package org.jenkinsci.plugins.workinghours.model;
+
+public class DateType {
+    public static String TYPE_GREGORIAN = "Gregorian";
+    public static String TYPE_CHINESE_LUNAR = "Chinese Lunar";
```

Comment:
We'll probably want to make these localizable, in case anyone decides to localize this plugin